### PR TITLE
Add a new page dedicated for working-groups 🍇

### PR DESCRIPTION
**Pull request by vdemeester**
- list all working groups,
- for each working groups: calendar, time, video call link, meetings
  notes and facilitators

I need to bootstrap some of the WG listed below :angel: and also, we should make sure I did put the right link at the right place :nerd_face: 

/hold

/cc @bobcatfish @kimsterv @abayer @dlorenc @chmouel @skaegi @ncskier @vtereso @dibyom 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: working-groups.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Tekton working group
+
+Most community activity is organized into _working groups_.
+
+Working groups follow the [contributing](./CONTRIBUTING.md) guidelines although
+each of these groups may operate a little differently depending on their needs
+and workflow.
+
+The working groups generate design docs which are kept in a
+[shared drive]()
```
Comment (chmouel) [inline working-groups.md]: no link, that's normal?

Comment (vdemeester) [reply working-groups.md]: > I need to bootstrap some of the WG listed below :angel: and also, we should make sure I did put the right link at the right place :nerd_face: 

Waiting for that :angel:

Comment (vdemeester) [reply working-groups.md]: (that's why the `/hold` :upside_down_face:)

Comment (chmouel) [reply working-groups.md]: you know that i never read descriptions 😝

**Comment by chmouel** [comment]
/lgtm 

@danielhelfand 🎉

File: contact.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ We are so happy you want to get in touch! You can reach and join us via:
 
 * [Slack](#slack)
 * [Our mailing list](#mailing-list)
-* [The Tekton working group](#working-groups)
+* [Calendar](#calendar)
+* [Shared Drive](#shared-drive)
```
Comment (bobcatfish) [inline contact.md]: :heart:

File: contact.md [hunk 3]
```diff
@@ -23,38 +24,18 @@ Some important channels:
...
+[working groups](./working-groups.md) meeting recordings!) are made visible to this
 group, so please join if you are interested in accessing those docs.
 
 Our mailing list for end users of Tekton is
 [tekton-users@](https://groups.google.com/forum/#!forum/tekton-users).
 
 ## Calendar
 
-Tekton events such as [working group](#working-group) and [other meetings](#other-meetings)
+Tekton events such as [working group](./working-groups.md) and [other meetings](#other-meetings)
```
Comment (bobcatfish) [inline contact.md]: a bit of scope creep but it looks like `#other-meetings` doesnt exist anymore

File: working-groups.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
+# Tekton working group
+
+Most community activity is organized into _working groups_.
+
+Working groups follow the [contributing](./CONTRIBUTING.md) guidelines although
+each of these groups may operate a little differently depending on their needs
+and workflow.
+
+The working groups generate design docs which are kept in a
+[shared drive](https://drive.google.com/drive/folders/0AFOvPxM9MpebUk9PVA)
```
Comment (bobcatfish) [inline working-groups.md]: do you think it makes sense to link to `#shared-drive` in process.md instead of duplicating this here? or move `#shared-drive` here?

File: working-groups.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+| Document Folder            | [Folder](https://drive.google.com/drive/folders/1QEj2TuIvxL6MvSSLwKucVOjpuVlydslP)                                                                                                                                          |
+| Slack Channel              | [#plumbing](https://tektoncd.slack.com/messages/plumbing)                                                                                                                                                                   |
+
+
+
+| &nbsp;                                                     | Facilitators      | Company | Profile                                     |
+| --------------------------------------------------------   | ----------        | ------- | ---------------------------------------     |
+| <img width="30px" src="https://github.com/vdemeester.png"> | Vincent Demeester | Red Hat | [vdemeester](https://github.com/vdemeester) |
+| <img width="30px" src="https://github.com/afrittoli.png">  | Andrea Frittoli   | IBM     | [afrittoli](https://github.com/afrittoli)  |
```
Comment (bobcatfish) [inline working-groups.md]: :wave: can i sign up as a facilitator?

File: working-groups.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+| Document Folder            | [Folder](https://drive.google.com/drive/folders/1dpGatrFh_Wykrs_tUkVK8hLNF1VaGHZd)                                                                                                                                                           |
+| Slack Channel              | [#docs](https://tektoncd.slack.com/messages/docs)                                                                                                                                                                                            |
+
+
+| &nbsp;                                                     | Facilitators      | Company | Profile                                     |
+| --------------------------------------------------------   | ----------        | ------- | ---------------------------------------     |
+| <img width="30px" src="https://github.com/vdemeester.png"> | Vincent Demeester | Red Hat | [vdemeester](https://github.com/vdemeester) |
```
Comment (bobcatfish) [inline working-groups.md]: i think @spomorski and/or @kimsterv might be interested in joining as facilitators also

File: working-groups.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+
+
+The current working groups are:
+
+- [General](#general)
+- [Triggers](#triggers)
+- [CLI](#cli)
+- [Documentation](#documentation)
+- [Productivity](#productivity)
+- [Beta APIs](#beta-apis)
```
Comment (bobcatfish) [inline working-groups.md]: i think this is called "beta release" down below

File: contact.md (hunk 3 above)
Comment (vdemeester) [reply contact.md]: oups :sweat_smile:

File: working-groups.md (hunk 4 above)
Comment (vdemeester) [reply working-groups.md]: It does !

File: working-groups.md (hunk 5 above)
Comment (vdemeester) [reply working-groups.md]: Yes :heart_eyes_cat:

File: working-groups.md (hunk 6 above)
Comment (vdemeester) [reply working-groups.md]: Right, I wasn't sure who to add :angel: forgot to comment on that :angel:

File: working-groups.md (hunk 7 above)
Comment (vdemeester) [reply working-groups.md]: :sweat_smile:

File: working-groups.md (hunk 6 above)
Comment (bobcatfish) [reply working-groups.md]: now it looks like you are on there twice ;)

Comment (bobcatfish) [reply working-groups.md]: :pray-panda:

Comment (vdemeester) [reply working-groups.md]: :man_facepalming: never edit a file too late :joy_cat:

Comment (bobcatfish) [reply working-groups.md]: XD

**Comment by vdemeester** [comment]
/hold cancel

I'll do the other changes (new WG) in follow-up PR
